content/challenges/medium/2 - Missing Number/func.js

```diff
@@ -10,5 +10,11 @@
  * @returns {Number} the missing number
  */
 module.exports = (arr) => {
-    // Your code here
+    for (i = 1; i <= arr.length + 1; i++) {
+        if (!arr.includes(i)) {
```

Comment:
How might you do this purely mathematically and not use includes?

Sometimes built in javascript functions are slower than doing something purely mathematically. It may be the case here because under the hood `.includes()` is likely looping through all the numbers each time you call it. So when you check 1 it loops arr + 1 times. Then 2 is another arr + 1 times.

Hint: We know that no matter what the numbers we are given won't repeat and are in a sequence. We can loop through one time get to the total number in the array passing in and then we can subtract that from what it SHOULD be.